workshops/views.py

```diff
@@ -375,6 +375,61 @@ def person_bulk_add_confirmation(request):
         return render(request, 'workshops/person_bulk_add_results.html',
                       context)
 
+@login_required
+def person_find_duplicates(request):
```

Comment:
I found a Django decorator to limit HTTP methods, which returns 405 for other methods - see https://github.com/neon-ninja/amy/commit/53abcd592e6696fa54be0b6d9914ca4f9cdd3ece